lib/nfg_ui/components/patterns/page_header.rb

```diff
@@ -56,12 +56,12 @@ def render
                     NfgUi::Components::Patterns::Media.new({}, view_context).render do
                       capture do
                         concat(NfgUi::Components::Elements::MediaObject.new({}, view_context).render {
-                          content_tag(:div, class: 'mr-2 h2') do
+                          content_tag(:div, class: 'mt-lg-1 mr-2 h3') do
                             NfgUi::Components::Foundations::Icon.new({ traits: [resource_theme_icon], class: "text-#{resource_theme_color} mr-0" }, view_context).render
                           end
                         })
                         concat(NfgUi::Components::Elements::MediaBody.new({}, view_context).render {
-                          content_tag(:h2) do
+                          content_tag(:h1) do
```

Comment:
@farhajawed the pageheader outputs new HTML as indicated here. So i think display app will need new HTML rendering.

Reply:
Looks like on public pages `page-header` class has a position of `fixed` for larger screen sizes. So, all the page headers are stacked up over the navbar, thereby hiding the navbar. I need some suggestion on how to tackle this. Should it be kept as it is?
![Screen Shot 2021-07-14 at 1 48 18 PM](https://user-images.githubusercontent.com/30298841/125691174-e06731f0-1b62-4bd0-bee0-fff9eed073eb.png)

Reply:
@kylejmorin do you think there's an inline css that could be added to this component for display purposes?

Reply:
@farhajawed , yes please add a `style = "position: relative;"` to each of those page headers so they display correctly in this app.